frontend/src/features/permits/helpers/dateSelection.ts

```diff
@@ -132,3 +140,18 @@ export const handleUpdateDurationIfNeeded = (
 
   return currentDuration;
 };
+
+/**
+ * Determine if start date or expiry date of permit applicationare in the past
+ * @param startDate Start date of the permit
+ * @param expiryDate Expiry date of the permit
+ * @returns True if either startDate or expiryDate are in the past
+ */
+export const isPermitStartOrExpiryDateInPast = (
+  startDate: Nullable<Dayjs>,
```

Comment:
It doesn't make sense to ask if a null/undefined startDate/expiryDate is in the past. It's better to check for actual non-null/non-undefined values instead (ie. ```startDate: Dayjs, expiryDate: Dayjs```)

Reply:
Eg. Start and expiry dates are null/undefined (possible due to still fetching), then in that case instead of calling this method for undefined values, simply not show the banner, or simply assume that the buttons are disabled, or something (since the information is not available)